README.md

```diff
@@ -168,6 +168,64 @@ class CardSection extends React.Component {
 export default CardSection;
 ```
 
+### Using the `PaymentRequestButtonElement`
+
+The [Payment Request Button](https://stripe.com/docs/elements/payment-request-button) lets you collect payment and address information from your customers using Apple Pay and the Payment Request API.
+
+To use the `PaymentRequestButtonElement` you need to first create a [`PaymentRequest` Object](https://stripe.com/docs/stripe.js#the-payment-request-object). You can then conditionally render the `PaymentRequestButtonElement` based on the result of `paymentRequest.canMakePayment` and pass the `PaymentRequest` Object as a prop.
```

Comment:
We downcase "object" (in PaymentRequest object and others) throughout our pages on <https://stripe.com/docs>.

In the longer term, I'm still bullish on moving the docs in this README into /docs for consistency.